Makefile

```diff
@@ -15,14 +15,16 @@ dependencies:
 	# pin version due to https://github.com/python-poetry/poetry/issues/7184
 	pip install poetry==1.2.2 --quiet --ignore-installed
 	poetry --version
-	poetry install --no-root
+	poetry install
 
 test:
 	coverage run -m pytest test/unit
-	coverage run --omit="test/*" -m unittest discover test.smoke
 
 integration:
-	coverage run --omit="test/*" -m unittest discover test.integration
+	coverage run -m pytest test/integration
```

Comment:
Ah interesting that we're running pytest via coverage... we should look into that in our other python repos, where we run coverage separately.